src/wx/cmdevents.cpp

```diff
@@ -2783,8 +2872,10 @@ EVT_HANDLER(ApplyPatches, "Apply IPS/UPS/IPF patches if found")
 
 EVT_HANDLER(MMX, "Enable MMX")
 {
+#ifdef MMX
```

Comment:
I'm not sure we should bother with this, every CPU for more than like two decades has MMX, we should just remove all references to this, but this can be done later.

Reply:
This is enabled only for x32 builds. Can we enable for all? I believe there should not be any issues with x64, but what about ARM and others? I have no idea.

Reply:
Yes x64 definitely has MMX, assuming there is no assembly involved, 32 bit assembly doesn't always work for x64 builds.

Reply:
And if there is assembly involved, it may not be better than what the compiler generates with the appropriate `-march` etc. flags, which we need to revisit at some point.